pkg/controller/jenkins/configuration/base/resources/pod.go

```diff
@@ -65,7 +65,7 @@ func GetJenkinsMasterContainerBaseEnvs(jenkins *v1alpha2.Jenkins) []corev1.EnvVa
 	envVars := []corev1.EnvVar{
 		{
 			Name:  "JENKINS_HOME",
-			Value: jenkinsHomePath,
+			Value: GetJenkinsHomePath(jenkins),
 		},
 	}
```

Comment:
Please delete those lines and add JENKINS_HOME in `NewJenkinsMasterContainer` function.

Reply:
The validation test on this fails as the unit test is being run against BaseEnvs

Reply:
So we now allow to override the `JENKINS_HOME` the test `override JENKINS_HOME env` can be deleted.

Reply:
Okay i will remove the validation accordingly.